v2/cn/ann-1019003001.md.tt2

```diff
@@ -28,6 +35,8 @@ A brife overview of the most noteable changes:
   recreate request buffer in balancer phase.
 * feature: supported receiveany on `ngx.req.socket(true?)` socks.
 * feature: implemented the new `ngx.ssl.server_port()` API to get server port.
+* Detect SSE4.2 support dynamically.
```

Comment:
We should make it clear that it is at runtime and is to support older Intel and AMD CPUs without the SSE 4.2 instruction set:

> Now we detect SSE 4.2 instruction set support at runtime in our LuaJIT to support older Intel and AMD CPUs without this instruction set with the same LuaJIT binary, otherwise binaries built on newer hardware would not work on older CPUs due to the "Invalid instruction" CPU exception.